chain.go

```diff
@@ -114,6 +114,24 @@ func (cc *Conn) AddChain(c *Chain) *Chain {
 	return c
 }
 
+// DelChain deletes the specified Chain. See also
+// https://wiki.nftables.org/wiki-nftables/index.php/Configuring_chains#Adding_base_chains
```

Comment:
Update this link to https://wiki.nftables.org/wiki-nftables/index.php/Configuring_chains#Deleting_chains

Reply:
done